lib/hanami/utils/json.rb

```diff
@@ -32,10 +58,26 @@ module Json
       # @raise [Hanami::Utils::Json::ParserError] if the paylod is invalid
       #
       # @since 0.8.0
+      #
+      # @deprecated Use {#parse} instead
       def self.load(payload)
+        Hanami::Utils::Deprecation.new("`Hanami::Utils::Json.load' is deprecated, please use `Hanami::Utils::Json.parse'")
```

Comment:
@AlfonsoUceda I want to get this security patch out ASAP. That means to bump to a patch version: from 0.9.0 to 0.9.0.

Until now we followed the rule to introduce breaking changes only on minor releases: eg from 0.9.0 to 0.10.0, but this doesn't justify that.

But that would require to update the Upgrade notes on the site and we publish those only when we bump Hanami version, which isn't the case.

---

That's the solution to release it as 0.9.1, without any breaking change, hence the deprecation warning. When we'll release 1.0.0.beta1 we can remove `.load`.